### PR TITLE
feat(armory.io): add docs.armory.io

### PR DESCRIPTION
**Pull request by natereid72**
removed references to beta, removed link to halyard config for git/repo artifact, added operator config for git/repo artifact.

Resolves GitHub issue: <link to GH issue>

Resolves Jira: [JIRAPROJECT-123]

### CODE REVIEW

File: content/en/docs/spinnaker-user-guides/kustomize-manifests.md [hunk 1]
```diff
@@ -74,50 +40,30 @@ Define the artifact:
...
-## Kustomize in 2.17 (Beta)
-​
+## Kustomize 
 ### Requirements
-Kustomize in 2.17+ requires the [git/repo](https://www.spinnaker.io/reference/artifacts/types/git-repo/) artifact type.
-​
-### Enable Kustomize
-​
-Kustomize can be enabled by a feature flag in Armory 2.16 and later.
+Kustomize requires the git/repo artifact type.
```
Comment (aimeeu) [inline content/en/docs/spinnaker-user-guides/kustomize-manifests.md]:
```suggestion
Kustomize requires the `git/repo` artifact type.
```

File: content/en/docs/spinnaker-user-guides/kustomize-manifests.md [hunk 2]
```diff
@@ -74,50 +40,30 @@ Define the artifact:
...
 
-**Operator**
+**Configure git/repo artifact with Operator**
 
-Add the following settings to the `SpinnakerService` manifest and apply the changes:
+Add the follwing settings to the `SpinnakerService` manifest:
 
 ```yaml
-apiVersion: spinnaker.armory.io/{{< param operator-extended-crd-version >}}
+apiVersion: spinnaker.armory.io/v1alpha2
```
Comment (aimeeu) [inline content/en/docs/spinnaker-user-guides/kustomize-manifests.md]:
```suggestion
apiVersion: spinnaker.armory.io/{{< param operator-extended-crd-version >}}
```

`param operator-extended-crd-version` is defined in config.toml so we don't have to hardcode `v1alpha2`

**Review by aimeeu — CHANGES_REQUESTED** [review]
Thanks for submitting this PR!

**Review by natereid72 — COMMENTED** [review]
ok

**Review by natereid72 — COMMENTED** [review]
ok

**Review by aimeeu — APPROVED** [review]

**Pull request merged**